var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,9 +42,7 @@ class Relion(CMakePackage, CudaPackage):
                     'Profiling', 'Benchmarking'))
 
     depends_on('mpi')
-    # relion will not build with newer versions of cmake
-    # per https://github.com/3dem/relion/issues/380
-    depends_on('cmake@3:3.9.4', type='build')
+    depends_on('cmake@2.8:', type='build')
```

Comment:
Can you confirm that both 3.0.7 and 3.0.8 can be built with newer versions of CMake? If not, we may need to do something like:
```python
depends_on('cmake@2.8:', type='build')
depends_on('cmake@:3.9.4', when='@:3.0.7', type='build')
```

Reply:
I was able to build them all using cmake 3.15 actually

I believe the previous cmake restrictions were due to cuda not actually Relion